burnman/anisotropy.py

```diff
@@ -0,0 +1,449 @@
+# This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences
+# Copyright (C) 2012 - 2015 by the BurnMan team, released under the GNU
+# GPL v2 or later.
+
+from __future__ import absolute_import
+import numpy as np
+import matplotlib.pyplot as plt
+
+class AnisotropicMaterial(object):
+    """
+    Base class for anisotropic materials
```

Comment:
I guess (for now?) ``Material`` and ``AnisotropicMaterial`` are completely unrelated?